packages/frontend/README.md

```diff
@@ -4,24 +4,6 @@
 
 This project assumes that you have a local Rafiki backend instance up and running. Typically, both `backend` and this project would be used within the local Rafiki playground. See the [local environment setup](../../localenv) for more information on how to set it up.
 
-To start the project in development mode, we first need to generate the GraphQL types for our queries and mutations.
-
-To generate the GraphQL types file run:
-
-```sh
-pnpm --filter backend generate
-```
-
-To start the application run:
-
-```sh
-pnpm --filter frontend dev
-```
-
-Now you can access the application on [http://localhost:3005](http://localhost:3005).
-
-> NOTE: When running Rafiki Admin in the development environment, it will connect to Cloud Nine Wallet Admin GraphQL - [http://localhost:3001/graphql](http://localhost:3001/graphql).
-
```

Comment:
I take it frontend cannot be run without kratos anymore and this is why this is removed? I do see an error when doing `pnpm --filter frontend dev`. 

```
Error: Environment variable kratosContainerPublicUrl is missing
    at /Users/blaircurrey/code/interledger/rafiki/packages/frontend/app/lib/envConfig.server.ts:8:11
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/Users/blaircurrey/code/interledger/rafiki/packages/frontend/app/lib/envConfig.server.ts:6:27)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Function.Module._load (node:internal/modules/cjs/loader:960:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:169:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:194:25)
```

I do think it's important to keep that working for development purposes so that we dont have to rebuild to see changes which not only takes ~20 seconds to rebuild but also wipes state.

Reply:
I suppose that would require not throwing an error when in dev and trying to read the kratos ENV vars and not finding them. And then when in dev and the kratos env var(s) are undefined we don't do any of the login, redirectIfUnauthorized, etc. Kinda sucks....